CONTRIBUTING.md

```diff
@@ -94,6 +94,14 @@ curl -X PUT "http://127.0.0.1:9200/cadence-visibility-dev"
 ```
 They will create an index template and an index in ElasticSearch. 
 
+:warning: Note: 
+You will run into error of `InternalServiceError{Message: ListClosedWorkflowExecutions failed, elastic: Error 400 (Bad Request): all shards failed [type=search_phase_execution_exception]}`
+if ElasticSearch index template was not created before using. As there will be a wrong index created by default. 
```

Comment:
>As there will be a wrong index created by default.
What does this mean?

Reply:
Oh this is a tricky one— I found quite some people forgot to apply the code ES schema before starting cadence server. By default ES would create an index(creating index on demand like Kafka topic, the fields are only based on the first write request) , which is wrong to use and would cause the error. 

However to fix this error , they can’t simply re-apply the ES schema. The index has to be deleted first before re-application.

Reply:
Added as a side note and added a "install-schema-es" to help prevent this.